soc/posix/inf_clock/Kconfig.soc

```diff
@@ -11,3 +11,13 @@ config SOC_POSIX
 	  sleep. Therefore do not use busy waits while waiting for something to happen
 	  (if needed use k_busy_wait()).
 	  Note that the interrupt handling is provided by the board.
+
+if SOC_POSIX
```

Comment:
This dependency would not be correct, or? Doesn't it depend just on ARCH_POSIX? (or maybe also in FLASH_NATIVE_POSIX)

Reply:
As it is part of inf_clock it did look more logic to make it depend on SOC_POSIX.
Further the FUSE support has no direct dependency on the flash device as it uses the file system API (``depends on FILE_SYSTEM``) to access the  flash device.

Reply:
But, the dependency would still be only on ARCH_POSIX? (not SOC_POSIX)

Reply:
changed dep to ARCH_POSIX (in subsys/fs/KConfig)